troposphere/static/js/components/common/InteractiveDateField.react.js

```diff
@@ -48,13 +48,31 @@ define(function (require) {
 
     render: function () {
       var labelEl = this.props.labelText ? (<label>{this.props.labelText}</label>) : "";
+      var style = {};
+      style.button = {cursor: "pointer"};
       return (
-        <div className="form-group">
-          {labelEl}
-          <input type='text' className='form-control' value={this.state.value} onChange={this.onValueChanged}/>
-          <span className="input-group-addon" id="enddate-set-addon" onClick={this.setEndDateNow}>Today</span>
-          <span className="input-group-addon" id="enddate-clear-addon" onClick={this.unsetDate}>Clear</span>
-        </div>
+            <div className="form-group">
+                    <label htmlFor="date-field">{labelEl}</label>
+                <div className="input-group">
+                    {/* Button, !button because bootstrap */}
```

Comment:
I believe `!button` is needed to block the cascade from bootstrap for the `button` selector/rules. Ultimately, this is a JSX comment (the `{/* ... */}`) - so including more context/description would be nice. 

The `labelEl` was added by me because there was no section-heading present before.